### DISCUSSION TITLE
Smaller images in MazGallery

### DISCUSSION THREAD

**Discussion by afonic** [LouisMazel/maz-ui#489] [Ideas]
Hello,

thanks for a great library! We love how trouble free is and how elegant everything looks!

I have a suggestion for MazGallery, it would be great if the thumbnails could be a different array set than the big images when the user clicks. The CMS we use automatically generates image presets so we ideally want to set the thumbnail size to something like 400px and only when the user clicks load the real 1920px image.

I'm not sure how you work with that project as it seems like an one person job, but we can even create a PR to get this started if you wish.

**Comment by LouisMazel** [comment]
Hello @afonic, 

This feature is now available in v3.18.0

Check out the documentation: https://louismazel.github.io/maz-ui-3/components/maz-gallery#types

You can now specify a thumbnail for each images.
